src/classes/CON_Email.cls

```diff
@@ -176,26 +178,29 @@ public class CON_Email {
 
             this.allFields = new List<CON_EmailField>();
             this.valuedFields = new List<CON_EmailField>();
-            Map<String, Schema.DescribeFieldResult> fieldMap = UTIL_Describe.getFieldsOfType('Contact', 'EMAIL');
```

Comment:
This seems like a safe enough change but let me know if its preferred to open a new PR/WI for this. When testing the bulk preferred change, there were a few intermittent unit test failures and I noticed this was inefficient.

Before the cache fix:
```
Class.hed.CON_Preferred_TDTM.run: line 56, column 1: public static void processPreferredEmail(Contact, List<Contact>): executed 1500 times in 6557 ms
Class.hed.CON_Email.processPreferredEmail: line 55, column 1: public hed.CON_Email.CON_EmailFieldList(Contact): executed 1500 times in 5714 ms
```

After the cache fix:
```
Class.hed.CON_Preferred_TDTM.run: line 56, column 1: public static void processPreferredEmail(Contact, List<Contact>): executed 1500 times in 1854 ms
Class.hed.CON_Email.processPreferredEmail: line 55, column 1: public hed.CON_Email.CON_EmailFieldList(Contact): executed 1500 times in 1139 ms
```

Reply:
I think it makes sense to have it as part of the same PR.